Source/RunActivity/Viewer3D/Signals.cs

```diff
@@ -466,9 +466,11 @@ public SignalTypeData(Viewer viewer, Orts.Formats.Msts.SignalType mstsSignalType
 
                         if (mstsSignalType.Semaphore)
                             glowDay = 0.0f;
+                        /* Jindrich
+                         * Why is this here? I want all my lights glowing, when I have glowing on
```

Comment:
As I see, you have created a different PR for this. To avoid merge issues, you should remove this line from this PR.

Probably this block should be kept for backwards compatibility anyway. I think you can manually force SHUNTING and INFO signals to glow using the sigcfg.

Reply:
This PR was cancelled.